main.py

```diff
@@ -51,9 +53,12 @@ def main():
         )
         distances_df = distances_df.rename("distances")
```

Comment:
What is the purpose of this line?

Reply:
distances_df is pandas.Series. In order to use merge:
```
distances_df = pd.merge(
            distances_df, breweries_df.beer_count, left_index=True, right_index=True, how="inner"
        )
```
Pandas series has to be named. Without renaming, following error occurs:
`ValueError: Cannot merge a Series without a name`

Reply:
We have a problem somewhere.
distances_df  is DataFrame
and your error message tells you that you cannot merge Series.

Reply:
Fixed